litemall-wx-api/src/main/java/org/linlinjava/litemall/wx/web/WxAddressController.java

```diff
@@ -183,4 +183,19 @@ public Object delete(@LoginUser Integer userId, @RequestBody LitemallAddress add
 		addressService.delete(id);
 		return ResponseUtil.ok();
 	}
+	
+	/**
+	 * 删除收货地址
+	 *
+	 * @param addressId  addressID
+	 * @return 删除操作结果
+	 */
+	@PostMapping("delete")
+	public Object delete(@RequestBody Integer addressId) {
```

Comment:
No need to use RequestBody when you just pass integer. If It is like USER.class, you can use post and requestBody. But just one param, DO NOT use POST and requestbody